### DISCUSSION TITLE
Why can't I remove BIT from my computer using npm uninstall?

### DISCUSSION THREAD

**Discussion by Olsha100** [teambit/bit#6113] [Help] [answered]
I have BIT  0.0.698 installed on my computer but I need to switch to 14.8.8 version .

In order to do that I try to uninstall 0.0.698 version BUT when I use a command:

`npm uninstall -g bit`

I get this:

```
up to date, audited 1 package in 315ms

found 0 vulnerabilities
```

When I try to find BIT in my global packages using:

`npm --location=global list`

BIT doesn't show up on the list.

Can anyone explain me what's going on ? Why can't I find BIT using npm list and remove it?

I use Windows 10

**Comment by toomanychrises** [comment, marked as answer]
use `npm uninstall -g @teambit/bvm` to remove bit

should produce this message: `removed 594 packages in 2s`

I actually had to go into `Users/bin` and `Users/.bvm` and delete them to completely remove bit

**Reply by StephenSHorton** [reply]
This should be marked as the answer 👍
